### PR TITLE
docs: add pinephone article

### PR DESCRIPTION
**Pull request by ghost**
Added an article about the Pinephone

### CODE REVIEW

File: content/blog/quick-guide-pinephone.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+What makes the Pinephone particulary interesting is that it is a pragmatic open
+hardware design. This makes it an excellent host for a Linux-based ecosystem.
+Linux kernel support and mobile interfaces like [Phosh](https://developer.puri.sm/Librem5/Software_Reference/Environments/Phosh.html)
+and [Plasma mobile](https://www.plasma-mobile.org/) are yet quite unstable;
+the main reason why the Pinephone is primarily for Linux enthousiasts.
+
+## Hardware update
+
+In january 2020 I ordered a Pinephone Braveheart(1.1 rev. hardware). This was a
+good excuse for me to get a bit [familiar]((https://gist.github.com/jvanveen/d395869f925605c09b0e5d1a0959b52b)
```
Comment (RubenHoms) [inline content/blog/quick-guide-pinephone.md]: Double `((` breaks formatting here.

File: content/blog/quick-guide-pinephone.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+To be honest; I find this the most boring part of reviewing the phone. Yes,
+I would like to use it as my daily mobile phone. My current phone doesn't do much
+besides calling, sms and running a week on one battery charge. Anyway, I've used
+the Pinephone for two days as my mobile phone and here are my findings:
+
+### Reachability - contacts, calls & sms
+
+I would like to have my contacts stored outside of the phone, so a phone
+crash won't wipe them all. I installed [Nextcloud](https://wiki.archlinux.org/index.php/Nextcloud)
+on a VPN and used the Nextcloud Contacts sync option in Gnome Contacts to sync
```
Comment (RubenHoms) [inline content/blog/quick-guide-pinephone.md]: Do you mean VPS instead of VPN here?

File: content/blog/quick-guide-pinephone.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+mode, so I'm not that worried about it. However, its good that see some
+awareness of the issue, before accidents happen. After a while, when
+unlocking the phone, the touch input seems to be lagging one event behind
+all the time, which forces me to reboot the phone.
+
+## Pocket-size Computer
+
+![Example image](/images/blog/convergence.jpeg)
+
+This is what I get really exciting about. Yes, the phone part is not 100% there
```
Comment (RubenHoms) [inline content/blog/quick-guide-pinephone.md]: Should be "get really excited about"

**Review by RubenHoms — CHANGES_REQUESTED** [review]
Awesome article! :+1:  I've heard some people at Spindle that are tinkering with the Pinephone as well, and for that purpose it seems really fit. Whatever the future might hold, the Pinephone really allows developers to see and try out the potential of an open (hardware & software) phone.

I added a few comments, but will get it merged once those are fixed :smile:

File: content/blog/quick-guide-pinephone.md (hunk 1 above)
Comment (ghost) [reply content/blog/quick-guide-pinephone.md]: Fixed

File: content/blog/quick-guide-pinephone.md (hunk 2 above)
Comment (ghost) [reply content/blog/quick-guide-pinephone.md]: Oops, yes indeed

File: content/blog/quick-guide-pinephone.md (hunk 3 above)
Comment (ghost) [reply content/blog/quick-guide-pinephone.md]: Check. Fixed

**Pull request merged**